### ISSUE TITLE
I-JEPA - contributing to lib

### ISSUE THREAD

**Issue by Natyren** [lightly-ai/lightly#1260]
In this issue, I have initiated the beginning of a PR to implement I-JEPA. If there are any concerns or suggestions regarding potential issues or best practices, please kindly share them.

**Comment by Natyren** [comment]
@guarin Hello! Can I implement additional module with Attention and Causal Attention (like in x-formers) for usage in this model (Causal Attention for Predictors).

**Comment by Natyren** [comment]
Sorry for interruption, looks like I can pass the attention mask to mha in PyTorch. So there is no need to add module

**Comment by Natyren** [comment]
Completed with experimental https://github.com/lightly-ai/lightly/pull/1273
